sources/mongodb/helpers.py

```diff
@@ -59,19 +57,32 @@ def make_query(self) -> TCursor:
 
     def clean_document(self, document: Dict[str, Any]) -> Any:
         """Clean document to be json serializable"""
-        output = {}
+        # Convert lists to json serializable
```

Comment:
this is a hack :) what you need to do is to recursively apply the simple mapping function recursively.
we have a util function: map_nested_in_place in dlt.common.util

the think that you need to write is a simple mapping that you have below and pass it:
```
            if isinstance(value, Decimal128):
                output[key] = value.to_decimal()
            elif isinstance(value, ObjectId):
            elif isinstance(value, (ObjectId, Decimal128)):
                output[key] = str(value)
else: return value

```